src/main/java/org/jenkins/plugins/lockableresources/queue/QueuedContextStruct.java

```diff
@@ -8,18 +8,24 @@
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 package org.jenkins.plugins.lockableresources.queue;
 
+import java.io.IOException;
 import java.io.Serializable;
 
+import com.infradna.tool.bridge_method_injector.WithBridgeMethods;
+import hudson.model.AbstractBuild;
+import hudson.model.Run;
 import org.jenkinsci.plugins.workflow.steps.StepContext;
-import org.jenkins.plugins.lockableresources.queue.LockableResourcesStruct;
 
 import edu.umd.cs.findbugs.annotations.Nullable;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
 
 /* 
  * This class is used to queue pipeline contexts 
  * which shall be executed once the necessary
  * resources are free'd.
  */
+@ExportedBean(defaultVisibility = 999)
```

Comment:
Why is this exported?

Reply:
unfamiliarity with the stapler code. These two that you questioned would have been left over from experiments and didn't break anything. I've removed them and everything seems to still work.